### PR TITLE
v644-2 2019 edit changes

### LINKED ISSUE #695: V644-2 2019 Changes
Update edit V644-2 for 2019 collection to include 3 as an acceptable response.  

If Sex of Applicant or Borrower equals 6, then Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname must equal 2 or 3.

### PR DESCRIPTION
**Pull request by MichaelFurey17**
Close #695

### CODE REVIEW

File: 2019/python/lar_constraints.py [hunk 1]
```diff
@@ -452,11 +452,11 @@ def v644_const(self, row):
 		"""1) If Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname equals 2,
 			then Sex of Applicant or Borrower must equal 1, 2, 3, or 6.
 		2) If Sex of Applicant or Borrower equals 6, then Sex of Applicant or Borrower Collected on the Basis of
-			Visual Observation or Surname must equal 2."""
+			Visual Observation or Surname must equal 2 or 3."""
 		if row["app_sex_basis"] == "2" and row["app_sex"] not in ("1", "2", "3", "6"):
 			row["app_sex"] = random.choice(("1", "2", "3", "6"))
 		if row["app_sex"] == "6":
-			row["app_sex_basis"] = "2"
+			row["app_sex_basis"] random.choice("2", "3")
```
Comment (Kibrael) [inline 2019/python/lar_constraints.py]: I only see a 2 listed in https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2019-hmda-fig.pdf.

Can you confirm the source of 2 and 3?

File: 2019/python/rules_engine.py [hunk 2]
```diff
@@ -1055,10 +1055,10 @@ def v644_1(self):
 	def v644_2(self):
 		"""An invalid Sex data field was reported.
 		2) If Sex of Applicant or Borrower equals 6, then Sex of Applicant or Borrower Collected on the Basis of
-		Visual Observation or Surname must equal 2."""
+		Visual Observation or Surname must equal 2 or 3."""
 		field = "Applicant Sex Basis"
 		edit_name = "v644_2"
-		fail_df = self.lar_df[(self.lar_df.app_sex=="6")&(self.lar_df.app_sex_basis!="2")]
+		fail_df = self.lar_df[(self.lar_df.app_sex=="6")&(~self.lar_df.app_sex_basis.isin(("2", "3")))]
```
Comment (Kibrael) [inline 2019/python/rules_engine.py]: What is the source of including 3? I can't find it in 2018 or 2019 FIG under v644

Comment (MichaelFurey17) [reply 2019/python/rules_engine.py]: This change is reflected in the last commit.

File: 2019/python/lar_constraints.py (hunk 1 above)
Comment (MichaelFurey17) [reply 2019/python/lar_constraints.py]: This change is reflected in the last commit.

**Pull request merged**